create_builder.go

```diff
@@ -8,15 +8,16 @@ import (
 	"github.com/buildpack/imgutil"
 	"github.com/pkg/errors"
 
-	"github.com/buildpack/pack/builder"
-	"github.com/buildpack/pack/dist"
-	"github.com/buildpack/pack/image"
-	"github.com/buildpack/pack/style"
+	builder2 "github.com/buildpack/pack/builder"
+	"github.com/buildpack/pack/internal/builder"
+	"github.com/buildpack/pack/internal/dist"
+	"github.com/buildpack/pack/internal/image"
+	"github.com/buildpack/pack/internal/style"
 )
 
 type CreateBuilderOptions struct {
 	BuilderName   string
-	BuilderConfig builder.Config
+	BuilderConfig builder2.Config
```

Comment:
:| Not a fan of `builder2`. Feel like if we are going to be running into this issue more frequently we should have a standard way of differentiating them.

Reply:
Agreed. That's definitely not intentional.